src/price-formatting/format.js

```diff
@@ -199,7 +199,10 @@ function formatPriceParts(numberFormatting, value, decimals, formatFlags, numera
 
     if (isNegative) {
         parts.Post = numberFormatting.negativePost;

```

Comment:
the same check needs to happen on post - thats for cultures where the - goes after or -1 is represented as (1)

Reply:
Adding check for post as well.